packages/explat-client-react-helpers/src/index.tsx

```diff
@@ -42,6 +54,7 @@ export default function createExPlatClientReactHelpers(
 			let isSubscribed = true;
 			exPlatClient.loadExperimentAssignment( experimentName ).then( ( experimentAssignment ) => {
 				if ( isSubscribed ) {
+					// The order here is important as React can rerender in between these two
 					setExperimentAssignment( experimentAssignment );
 					setIsLoading( false );
```

Comment:
Ah, actually I am not even sure the order is guaranteed.

I should put them both in the same `useState` so the change will be atomic.

Reply:
It's now atomic